gensim/models/base_any2vec.py

```diff
@@ -130,7 +130,21 @@ def _do_train_epoch(self, corpus_file, thread_id, offset, cython_vocab, thread_p
         raise NotImplementedError()
 
     def _do_train_job(self, data_iterable, job_parameters, thread_private_mem):
-        """Train a single batch. Return 2-tuple `(effective word count, total word count)`."""
+        """Train a single batch.
+        `
```

Comment:
I think we should document the input parameters as well.

Reply:
I agree, though there are no clear constraints on what the inputs actually are. They can actually be pretty much anything depending on how the derived class implements `_get_thread_working_mem` and `_get_job_params`, the duty is to the derived class to figure out what to do with them. I'll figure out something.